### ISSUE TITLE
Please help!

### ISSUE THREAD

**Issue by zutmkr** [siriuslysam/light_swiper#1]
@siriuslysam 
Hi! 
Sorry for the spam, but github doesn't have any form of private messaging.

I am referring to issue you submmited on VS code:

https://github.com/microsoft/vscode-remote-release/issues/2145

I run on the same issue. Did you found a solution? I am runnig raspbian on Virtualbox, and cant connect remotely with Code server. The same "/node not found" error. But the file is there.

Did you manage to solve this?
Thanks!

**Comment by siriuslysam** [comment]
Hello @zutmkr 

So I've tested the remote ssh feature of Code with an actual RaspberryPi, in which case worked, whereas it didn't on the virtual version via VMware. My hunch is that the different architectures; RPi uses ARM whereas the virtual machine is running off of your desktop's processor which is not ARM. The node/libs used for the Remote-SSH for the RPi are expecting an ARM processor, thus will not work on the virtual machine.

TL;DR Not solved due to System Requirements listed under [https://code.visualstudio.com/docs/remote/ssh](url)

**Comment by zutmkr** [comment]
@siriuslysam

That is correct. My wife also pointed me on the same answer (she's a programmer too :d).

Thank you for answer and have a nice day!
I think we can resolve the issue :)
Bye!
